data/map planets.txt

```diff
@@ -1099,6 +1099,12 @@ planet "Dlia Jzaur"
 	description `	Oddly enough, Dlia Jzaur has many unnatural looking formations on the surface that seem to reach up to the sky in areas where eclipses often occur. The formations are hollow underneath, but show no signs of Gegno involvement.`
 	government Uninhabited
 
+planet "Donya"
+	attributes uninhabited
+	landscape land/fields29
+	description `Donya is a lush, green, Earth-like world filled with vegetation and growing matter. With clouds and a blue sky, it was idyllic to the eyes of early settlers. Despite that, it remains unsettled as rains of poisonous acid fall on regular timeframes, rendering the planet uninhabitable to human life. The foliage planted by the initial wave of colonists seems to have adjusted to the terrible rain quite well unlike their masters, who left at the first opportunity.`
```

Comment:
Okay, so it's filled with growing matter but simultaneously has a bad case of acid rain? That doesn't make any sense. Generally, any acid that is dangerous to humans is also dangerous to plants and animals. Is there some sort of radiation or something causing the foliage to mutate, creating a monoculture, and also increasing the risk to humans? Was it super genetically engineered before it was planted, and supposed to scrub the acid, but failed to do so?

Reply:
See below. Also I figured that the trees were better at getting nutrients from the soil...
But probably genetically engineered.

Reply:
Does this look better?
Might be too long though.
```suggestion
	description `Donya is a lush, green, Earth-like world filled with vegetation and growing matter. With clouds and a blue sky, it was idyllic to the eyes of early settlers who were being funded by billionaire philanthropists.`
	description `	Despite the beauty and the funding, it remains unsettled, as rains of poisonous acid fall on regular timeframes, rendering the planet uninhabitable to human life. The foliage planted by the initial wave of colonists to scrub the skies of the rain seems to have adjusted to the terrible liquid quite well, although it hasn't managed to entirely eliminate it. The planters, who couldn't bear the dreadful weather, left when it was discovered that their plan would not work.`
```

Reply:
Pretty good, although a bit wordy in places.
```suggestion
	description `Donya is a lush, green, Earth-like world filled with vegetation and growing matter. With clouds and a blue sky, it was idyllic in the eyes of early settlers and the billionaire philanthropists who funded them.`
	description `	Despite the beauty and the funding, it remains unsettled, as rains of poisonous acid fall on regular timeframes, rendering the planet uninhabitable to human life. The foliage planted by the initial wave of colonists to scrub the skies of the rain seems to have adjusted to the terrible liquid quite well, although it hasn't managed to entirely eliminate it. The planters, who couldn't bear the dreadful weather, left when they discovered that they couldn't stop the rains.`
```